### PR TITLE
fixing according dcterm specs.

### PR DESCRIPTION
**Pull request by zyuhel**
According to dcterms specs, there are no author field, it is named as creator - http://dublincore.org/documents/usageguide/elements.shtml#creator

May be in the wild someone use dcterms:author then i will modify it to check if value is isset;

### CODE REVIEW

**Comment by oscarotero** [comment]
Thank you for your contribution.
Sadly, not all sites respect the spects, so I suggest to check both values (author and creator).

**Comment by zyuhel** [comment]
Interesting why soundcloud tests are failing on travis while they are normal on local machine.

File: src/Providers/Dcterms.php [hunk 1]
```diff
@@ -51,7 +51,10 @@ public function getDescription()
      */
     public function getAuthorName()
     {
-        return $this->bag->get('author');
+        $author=$this->bag->get('creator');
+        if (!empty($author))
+            return $this->bag->get('creator');
+        return $this->bag->get('author');
```
Comment (oscarotero) [inline src/Providers/Dcterms.php]: This code is shorty and allow to be extended with more possible values:
```php
return $this->bag->get('creator') ?: $this->bag->get('author');
```

**Comment by oscarotero** [comment]
Yes, testing is a nightmare due things like this :)

**Comment by oscarotero** [comment]
Thanks :+1:

**Pull request merged**